common/entity/ExpertData.ts

```diff
@@ -49,7 +50,7 @@ export class ExpertData {
     active: boolean;
 
     @Column({
-        default: false
+        default: ExpertAllowedIndication.PENDING
     })
-    allowed: boolean;
+    allowed: ExpertAllowedIndication;
```

Comment:
In the past I have forgotten this a few times too, so I want to mention it here because here I just noticed it: Enum defines without explicitly specifying the enum type were saved as a string in the database, and not as a Postgres Enum, which would be preferred, I guess. Also see documentation [here](https://github.com/typeorm/typeorm/blob/master/docs/entities.md#enum-column-type).